apps/next/src/popup/app.tsx

```diff
@@ -19,6 +20,25 @@ import { Onboarding } from '@/pages/Onboarding';
 import { MdSwitchAccount } from 'react-icons/md';
 import { Route, Switch, useHistory } from 'react-router-dom';
 
+import { Providers } from '.';
+import { Header } from '@/components/Header';
+
+const pagesWithoutHeader = [
+  '/tokens/manage',
+  '/bridge/confirm',
+  '/bridge/transaction-status',
+  '/bridge/transaction-details',
+  '/send/confirm',
+  '/collectible',
+  '/collectible/send/confirm',
+  '/accounts',
+  '/import-private-key',
+  '/import-with-walletconnect',
+  '/defi',
+  '/fireblocks',
+  '/export-private-key',
+];
```

Comment:
These URLs do not exist at the moment, let's not tie ourselves to them in the new app.